doc/themes/scikit-learn/layout.html

```diff
@@ -149,8 +149,8 @@
 			     style="max-height: 200px; max-width: 629px; margin-left: -21px;"></div></a>
 		  </div>
 		  <div class="item">
-		    <a href="{{pathto('auto_examples/covariance/plot_outlier_detection') }}">
-		      <img src="_images/sphx_glr_plot_outlier_detection_003_carousel.png"></a>
+		    <a href="{{pathto('auto_examples/plot_anomaly_comparison') }}">
+		      <img src="_images/sphx_glr_plot_anomaly_comparison_001.png"></a>
```

Comment:
`_carousel` or `_thumb` missing at the end of the name. I don't really know the difference...

Reply:
Ah.. I will fix that.
Actually website works good. but the image size is too big. I will change a thumb image.